OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -260,4 +260,4 @@ public bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, ref
 			public bool IsQueued { get; protected set; }
 		}
 	}
-}
+}
```

Comment:
Please keep the EOF newlines (affects multiple files)

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file